src/main/java/org/bookstore/exceptions/GlobalExceptionHandler.java

```diff
@@ -35,16 +35,22 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
         return new ResponseEntity<>(body, headers, status);
     }
 
-    @ExceptionHandler(value = {EntityNotFoundException.class})
+    @ExceptionHandler(EntityNotFoundException.class)
     public ResponseEntity<Object> handleEntityNotFound(EntityNotFoundException ex) {
         return new ResponseEntity<>(ex.getMessage(), HttpStatus.NOT_FOUND);
     }
 
-    @ExceptionHandler(value = {DataProcessingException.class})
+    @ExceptionHandler(DataProcessingException.class)
     public ResponseEntity<Object> handleDataProcessingException(DataProcessingException ex) {
         return new ResponseEntity<>(ex.getMessage(), HttpStatus.UNPROCESSABLE_ENTITY);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    protected ResponseEntity<String> handleRegistrationExceptions(RegistrationException ex) {
+        return new ResponseEntity<>("registration-exception: "
```

Comment:
this suffix is unnecessary
```suggestion
        return new ResponseEntity<>(
```